ipamd/rpc_handler.go

```diff
@@ -47,9 +47,24 @@ func (s *server) AddNetwork(ctx context.Context, in *pb.AddNetworkRequest) (*pb.
 		Name:      in.K8S_POD_NAME,
 		Namespace: in.K8S_POD_NAMESPACE,
 		Container: in.K8S_POD_INFRA_CONTAINER_ID})
+
+	resp := pb.AddNetworkReply{Success: err == nil, IPv4Addr: addr, IPv4Subnet: "",
+		DeviceNumber: int32(deviceNumber), UseExternalSNAT: s.ipamContext.networkClient.UseExternalSNAT()}
+
+	vpcCIDRs := s.ipamContext.awsClient.GetVPCIPv4CIDRs()
+
+	var pbVPCcidrs []string
+
+	for _, cidr := range vpcCIDRs {
+		log.Debugf("VPC CIDR %s", cidr)
+		pbVPCcidrs = append(pbVPCcidrs, *cidr)
+	}
+
+	resp.VPCcidrs = pbVPCcidrs
```

Comment:
If you get the pbVPCcidrs before creating the response, it's a bit easier to read. Something like
```
	vpcCIDRs := s.ipamContext.awsClient.GetVPCIPv4CIDRs()

	var pbVPCcidrs []string
	for _, cidr := range vpcCIDRs {
		pbVPCcidrs = append(pbVPCcidrs, *cidr)
	}

	resp := pb.AddNetworkReply{
		Success:  err == nil,
		IPv4Addr: addr, IPv4Subnet: "",
		DeviceNumber:    int32(deviceNumber),
		UseExternalSNAT: s.ipamContext.networkClient.UseExternalSNAT(),
		VPCcidrs:        pbVPCcidrs}
```

Reply:
done

Reply:
Almost.. 😛 `resp.VPCcidrs = pbVPCcidrs` could be moved in. 😁